include/osquery/config.h

```diff
@@ -484,6 +484,10 @@ class ConfigParserPlugin : public Plugin {
   /// Allow parsers to perform some setup before the configuration is loaded.
   Status setUp() override;
 
+  Status call(const PluginRequest&, PluginResponse&) override {
```

Comment:
NB: The registry concept is forcing registry "types" like Config, Logger, Table to implement a `call` method.